src/hooks/useLineageFilter.js

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react'
+
+import useQueryAsState from './useQueryAsState'
+
+// https://personal.sron.nl/~pault/
+const tolMutedQualitative = [
```

Comment:
Is there a circumstance where these get used, given that we supply default colours in DataLoader? (I'm fine with overwriting DataLoader colours with these).

Reply:
ah right, I forgot about DataProvider, I'll move these there 👍🏽